src/loaders/geoJsonUrlLoader.js

```diff
@@ -21,49 +26,20 @@ const fetchData = async (url, engine, instanceBaseUrl) => {
                     ? JSON.parse(await data.geojson.text()) // TODO - remove once Blob fix implemented in app-runtime (LIBS-542)
                     : data.geojson
             )
-            .catch((e) => {
-                if (typeof e === 'object' && e.details?.message) {
-                    if (
-                        e.details.message.toLowerCase().includes('jwt expired')
-                    ) {
-                        throw new Error(
-                            i18n.t('Layer authorization is no longer valid.')
-                        )
-                    } else if (
-                        e.details.message.toLowerCase().includes('not found')
-                    ) {
-                        throw new Error(i18n.t('Url to geojson was not found.'))
-                    }
-
-                    throw new Error(e.details.message)
-                }
-
-                throw new Error(e)
+            .catch(() => {
+                throw new Error('Error')
```

Comment:
This looks strange to me. Can you explain how this will be shown to the user? And the purpose of passing `'Error'` (instead of just `new Error()`)